README.md

```diff
@@ -12,17 +12,44 @@ The Azure [validator](https://github.com/spectrocloud-labs/validator) plugin ens
 
 The Azure validator plugin reconciles `AzureValidator` custom resources to perform the following validations against your Azure environment:
 
-TODO
+1. Compare the Azure RBAC permissions associated with a [security principal](https://learn.microsoft.com/en-us/azure/role-based-access-control/overview#security-principal) against an expected permission set.
 
 Each `AzureValidator` CR is (re)-processed every two minutes to continuously ensure that your Azure environment matches the expected state.
 
 See the [samples](https://github.com/spectrocloud-labs/validator-plugin-azure/tree/main/config/samples) directory for example `AzureValidator` configurations.
 
-(TODO: actual samples)
+## Authn & Authz
 
-## Supported Service Quotas by Azure Service
+Authentication details for the Azure validator controller are provided within each `AzureValidator` custom resource.
 
-TODO
+Supported authentication methods:
+
+* Implicit
+  * Plugin is authenticated by [Workload identity](https://learn.microsoft.com/en-us/azure/aks/workload-identity-overview)
+  * To use this method:
+    1. Set Helm value `AzureValidator.auth.implicit` to `true`.
+    1. Ensure that Workload identity is set up for your AKS cluster, including the [managed identity](https://learn.microsoft.com/en-us/azure/aks/workload-identity-overview) and [federated identity credential](https://learn.microsoft.com/en-us/azure/aks/workload-identity-overview).
+    1. Set Helm value `AzureValidator.auth.serviceAccountName` to the name of the Kubernetes ServiceAccount referenced in the federated identity credential. The Helm chart will create the ServiceAccount for you.
+    1. Set Helm value `AzureValidator.controllerManager.serviceAccount.annotations` such that the `azure.workload.identity/client-id` annotation is set to the Client ID of the Microsoft Entra ID managed identity.
```

Comment:
Note that this is not good enough for implicit auth. We're missing the part where we tell the user how to override which manager Pod labels the Helm chart will use so that workload identity works. And we have to add support for that to the Helm chart first.

Alternatively, we remove support for implicit auth for now til we sort that out.

Reply:
As discussed, we require the user to have created the ServiceAccount already if they're specifying its name here. So we can allow them to follow Azure docs and make sure things are in order.